tests/rptest/scale_tests/tiered_storage_stability_test.py

```diff
@@ -209,3 +212,85 @@ def stage_decommission(self):
         waiter = NodeDecommissionWaiter(self.redpanda, node_id, self.logger)
         waiter.wait_for_removal()
         self.redpanda.stop_node(node)
+
+
+    @cluster(num_nodes=5, log_allow_list=NOS3_LOG_ALLOW_LIST)
+    def test_with_cloud_storage(self):
```

Comment:
nit: maybe call this test_disrupt_cloud_storage or something? Both tests test cloud storage